moduleroot/Gemfile

```diff
@@ -144,6 +144,16 @@ if explicitly_require_windows_gems
   gem "win32-taskscheduler", "0.2.2",   :require => false
   gem "windows-api", "0.4.3",           :require => false
   gem "windows-pr",  "1.2.3",           :require => false
+else
+  if puppet_gem_location == :gem && Gem::Platform.local.os == 'mingw32'
```

Comment:
`puppet_gem_location = :gem` - I would have assumed we were hitting this already.